ui-admin/src/test-utils/mocking-utils.tsx

```diff
@@ -57,6 +57,28 @@ export const mockSurvey: () => Survey = () => ({
   createdAt: 0
 })
 
+/** returns a list of survey versions */
+export const mockSurveyVersionsList: () => Survey[] = () => ([
+  {
+    id: 'surveyId1',
+    stableId: 'survey1',
+    version: 1,
+    content: '{}',
+    name: 'Survey number one',
+    lastUpdatedAt: 0,
+    createdAt: 0
+  },
+  {
+    id: 'surveyId1',
+    stableId: 'survey1',
+    version: 2,
+    content: '{}',
+    name: 'Survey number one',
+    lastUpdatedAt: 0,
+    createdAt: 0
+  }
+])
```

Comment:
Using the existing mockSurvey() function will make it clearer which parameters you care about in this list. 

```suggestion
export const mockSurveyVersionsList: () => Survey[] = () => ([
  {
     ...mockSurvey(),
     id: 'surveyId1',
     stableId: 'survey1',
     version: 1
  },
  {
     ...mockSurvey(),
     id: 'surveyId2',
     stableId: 'survey1',
     version: 2
  }
])
```